pylint_odoo/checkers/modules_odoo.py

```diff
@@ -811,3 +817,18 @@ def _check_file_not_used(self):
         if self.msg_args:
             return False
         return True
+
+    def _check_xml_attribute_translatable(self):
+        """The xml attribute is missing the translation="off" tag
+            Example  <attribute name="groups">sale.group</attribute>
+        """
+        self.msg_args = []
+        for xml_file in self.filter_files_ext('xml', relpath=True):
+            doc = self.parse_xml(os.path.join(self.module_path, xml_file))
+            if type(doc) is lxml.etree._ElementTree:
```

Comment:
...or What about use a sentence similar to https://github.com/Vauxoo/pylint-odoo/blob/9762664/pylint_odoo/checkers/modules_odoo.py#L735-L737?

```python
            doc = self.parse_xml(os.path.join(self.module_path, xml_file))
            openerp_nodes = doc.xpath("/openerp") \
                if not isinstance(doc, basestring) else []
```